Objects/object.c

```diff
@@ -2389,6 +2386,10 @@ new_reference(PyObject *op)
 #ifdef Py_TRACE_REFS
     _Py_AddToAllObjects(op);
 #endif
+    if (_PyRuntime.ref_tracer.tracer_func != NULL) {
+        void* data = _PyRuntime.ref_tracer.tracer_data;
+        _PyRuntime.ref_tracer.tracer_func(op, PyRefTracer_CREATE, data);
+    }
```

Comment:
Micro-optimize access to _PyRuntime, since this function is performance critical:

```suggestion
    struct _reftracer_runtime_state *tracer = &_PyRuntime.ref_tracer;
    if (tracer->tracer_func != NULL) {
        void* data = tracer->tracer_data;
        tracer->tracer_func(op, PyRefTracer_CREATE, data);
    }
```

Python is usually built with -fPIC, global variables are addressed via the instruction pointer which is more complicated. It's not the case with my micro-optimization.

Same in _Py_Dealloc().